src/configupdater/section.py

```diff
@@ -133,6 +133,13 @@ def __deepcopy__(self: S, memo: dict) -> S:
         return clone._copy_structure(self._structure, memo)
 
     def optionxform(self, optionstr: str) -> str:
+        """Delegates :meth:`~configupdater.document.Document.optionxform`
+        to its parent container.
+
+        Please notice that when the option object is :obj:`detached
+        <configupdater.block.Block.detach>`, this method will simply return
+        ``optionstr`` as it is, without any changes.
+        """
         if self.has_container():
             return self.document.optionxform(optionstr)
         return optionstr.lower()
```

Comment:
Here `optionstr` is still converted to lower although it says otherwise in the docstring? And different to `optionxform` in `Option`?

Reply:
Oops, sorry, I forgot to change :P 

Thanks @FlorianWilhelm